ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -843,14 +849,24 @@ std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optional<ttnn::T
         }
         conv_config.deallocate_activation = true;
     }
+    auto output_parallel_config = parallel_config;
+    if(conv_config.shard_layout == ttnn::TensorMemoryLayout::WIDTH_SHARDED)
```

Comment:
shard_layout is std::optional so it might not be present here

Reply:
Fixed.